docs/geode-native-docs/introduction/quickstart.html.md.erb

```diff
@@ -53,7 +53,7 @@ for the latest Java version for your operating system. See the installation info
 Geode User's Guide for the versions of Java that are compatible with Geode. 
 
 - **Security Toolkit** (optional): The QuickStart security examples rely on a security plugin, `securityImpl`, which is based on the OpenSSL toolkit.
-If you plan to run the security examples, download the latest version of release 1.0.1 (must be 1.0.1) from the [OpenSSL website] (http://www.openssl.org/source/).
+If you plan to run the security examples, download the latest version of release 1.0.2 (must be 1.0.2) from the [OpenSSL website] (http://www.openssl.org/source/).
```

Comment:
For maintenance purposes it seems cleaner to have a single section in our docs that talk about external requirements like OpenSSL and then have all the places talk about that requirement to reference back to that page. This way when we change versions, instructions, etc., we don't have to change multiple places in the docs.